js_modules/dagit/packages/core/src/runs/RunRoot.tsx

```diff
@@ -130,13 +130,6 @@ const RUN_ROOT_QUERY = gql`
       __typename
       ... on Run {
         id
-        pipeline {
-          __typename
-          ... on PipelineReference {
-            name
-            solidSelection
-          }
-        }
         ...RunFragment
```

Comment:
just for hygiene, can we add `pipelineName` here outside of the RunFragment splat?